.github/workflows/ci.yaml

```diff
@@ -1,140 +1,141 @@
 name: ci
 on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+        env:
+            IS_CI: true
```

Comment:
I don't understand why there is that big of a diff on this file, can you explain the changes?

Reply:
this is caused by formatting change, l can revert this if you want to

Reply:
No, it's fine, just to confirm that only this env have changed.  

Would you apply the env only to the task that needs it tho.  
I prefer this to a global, with a global we can't tell why it's important.

Reply:
l see, sure not a problem!